clusterloader2/pkg/state/state.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package state
+
+import (
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+// TestState contains actual state of the running test
+// It has information about objects in namespaces,
+// currently running measurements and available tuning sets.
+type TestState struct {
+	namespaces *namespacesState
```

Comment:
Please give this more apt name, maybe `namespacesState`?

Reply:
(if golang allows)

Reply:
renamed to namespacesState.